OpenSim/Simulation/Manager/Manager.h

```diff
@@ -148,7 +148,7 @@ class OSIMSIMULATION_API Manager
      * must call setIntegrator() on your own. You should use one of the other
      * two constructors. */
     DEPRECATED_14("There will be no replacement for this constructor.")
```

Comment:
Maybe indicate the use of `Manager(Model& )` or `Manager(Model&, Integrator&)` instead?